.circleci/config.yml

```diff
@@ -1,70 +1,129 @@
-version: 2
+version: 2.1
+
+defaults: &defaults
```

Comment:
I think you could update this to be a reusable executor:
https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-executors

Reply:
What's the difference?

Reply:
Not a lot in this instance.  But using the executor you could overwrite the fields set by the executor if needed.  I was just throwing it out as an option mostly because I kind of like the look of setting the executors rather than the Yaml anchor.

Reply:
If you like the look, I'll roll the dice and update this.